actionpack/lib/action_controller/metal/request_forgery_protection.rb

```diff
@@ -111,9 +111,9 @@ module ClassMethods
       #     protect_from_forgery except: :index
       #   end
       #
-      # You can disable forgery protection on controller by skipping the verification before_action:
+      # You can disable forgery protection on a controller using:
       #
-      #   skip_before_action :verify_authenticity_token
+      #   skip_forgery_protection
```

Comment:
Since this is just a bare method call, what about adding some surrounding context?

```suggestion
      #   class BarController < ApplicationController
      #     skip_forgery_protection
      #   end
```

Reply:
Much better!